package-lock.json

```diff
@@ -4782,12 +4782,9 @@
       }
     },
     "caniuse-lite": {
-      "version": "1.0.30001260",
-      "resolved": "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001260.tgz",
-      "integrity": "sha512-Fhjc/k8725ItmrvW5QomzxLeojewxvqiYCKeFcfFEhut28IVLdpHU19dneOmltZQIE5HNbawj1HYD+1f2bM1Dg==",
-      "requires": {
-        "nanocolors": "^0.1.0"
-      }
+      "version": "1.0.30001332",
```

Comment:
From time to time we need to run `npx browserslist@latest --update-db`. This is the result.